doc/user_guide/user_guide_107.md

```diff
@@ -0,0 +1,143 @@
+# Exasol Power BI Connector 1.0.7 Release - User Guide
+
+## New features
+
+- Custom Query support
```

Comment:
But the connector is also bundeled with power bi right?
Maybe you can tell here somewhere that it's just for the advanced users, and defualt users can pic the built-in connector

Reply:
it's for limited use